.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+<!-- Please search existing issues to avoid creating duplicates. -->
+<!-- You can search for more info on https://discuss.dgraph.io too. -->
+<!-- Also please test using the latest Dgraph Version. -->
+<!-- Thank you for being part of Dgraph Community, cheers. -->
+
+**Desktop (please complete the following information):**
+ - OS: <!-- [e.g. Linux + distro + kernel] -->
+ - Browser: <!-- [e.g. chrome, safari] -->
+ - Dgraph Version: <!-- [e.g. v1.0.7-RC4] -->
+ - RAM available: <!-- [e.g. 32GB with "lru_mb 2048" for each Dgraph Server instance ]  -->
+ - Dgraph Client: <!-- [e.g. Dgo, Dgraph-js, dgraph4j, pydgraph] -->
+
+<!--  If you are not using Containerization, please delete this question snippet -->
+**if Containerization (please complete the following information):**
+ - Main Container Software: <!-- [e.g. Docker v18.06.1-ce-rc1, kubernetes v1.11.2, other] -->
+ - Integration (if applicable):  <!-- [e.g. AWS details, GKE details, Minikube, Docker toolbox, Virtual Machine details if applicable, other type of integration] -->
+ - Dgraph TAG Version: <!-- [e.g. v1.0.7-RC4] -->
+<!--  If you are not using Containerization, please delete this question snippet above -->
+
+**Describe the bug**
+<!-- A clear and concise description of what the bug is. -->
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Command/config used to run Dgraph. 
+2. Exec '....'
+3. "Ratel UI shows" '....'
+4. See error
+
+**Expected behavior**
+<!-- A clear and concise description of what you expected to happen. -->
+
+**Screenshots**
+<!-- If applicable, add screenshots to help explain your problem. -->
+
+**Additional context**
+- Have you tried reproducing the issue with latest release?
```

Comment:
Move this up. This is less of a question, and more of a suggestion that they should do this.